Docker_files/ocsci_container/README.md

```diff
@@ -0,0 +1,54 @@
+# ocs-ci Containers for OpenShift Data Foundation
+
+This project builds containers for [ocs-ci].
+The primary purpose for doing so was to be able to run [ocs-ci] tests with container
+
+The containers are expected to run with a service account that has admin credentials.
+
+### Build Image
+
+**Change directory to Docker_files/ocsci_container [$ cd Docker_files/ocsci_container]**
+
+**docker/podman build -t <image-name> -f Dockerfile_ocsci . --build-arg BRANCH_ID_ARG=<branch-id/master/stable>**
+
+BRANCH_ID_ARG:
+There are 3 options:
+1. BRANCH_ID_ARG is empty -> checkout to stable branch
+2. BRANCH_ID_ARG=master -> checkout to master branch
+3. BRANCH_ID_ARG=PR-ID [1234] -> checkout to relevant branch based on pr-id
+
+### Run Container
+```commandline
+docker/podman run -v <kubeconfig-path>:/opt/cluster <image-name> run-ci --cluster-path /opt/cluster_path
```

Comment:
You can directly mount volume to `/opt/cluster_path` then you don't need to do any cp and so on.
```suggestion
docker/podman run -v <kubeconfig-path>:/opt/cluster_path <image-name> run-ci
```

I think that --cluster-path /opt/cluster_path should not be passed by user here as it should be static and handled in entry point.

Reply:
We need to copy the kubeconfig to `auth` folder

Reply:
Why, if you mount whole cluster dir than you already have auth folder in there

Reply:
`<kubeconfig-path>:/opt/cluster_path` we can change it to ` <kubeconfig-path>:/opt/cluster_path/auth`
I dont want the customer copy the kubeconfig to `auth` on local machine .
`run-ci --cluster-path /opt/cluster_path .....` 

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L18

What do you think?

Reply:
done

Reply:
`--cluster-path /opt/cluster_path`  should be removed here.

Just describe in doc that `--cluster-path` should not be passed and this will be automatically added to provided and mandatory volume mounted to container.